src/tools/sdk/_windows.md

```diff
@@ -1,11 +1,13 @@
+How to install?
+
 You can install the Dart SDK using [Chocolatey.][Chocolatey]
 
-{{site.alert.important}}
-These commands require administrator privileges.
-If you need help on starting an administrator-level command prompt,
-try a search like
-<em><a href="https://www.google.com/search?q=cmd+admin"
-  target="blank">cmd admin</a>.</em>
+{{site.alert.important}} These commands require administrative privileges. 
+To start command prompt as an administrator following these steps:
+1. Press <kbd>Windows+R</kbd> to open the **Run** window. 
+2. Type “cmd” into the box. 
+3. Press <kbd>Ctrl+Shift+Enter</kbd> to run the command as an administrator.
+
 {{site.alert.end}}
```

Comment:
I'm thinking since there are multiple ways to run commands as an administrator, through the old PowerShell and command prompt UIs as well as with the new Windows Terminal, we can present this as one example. I've also added some extra formatting. How does this look?

```suggestion
{{site.alert.important}} 
  These commands require administrative privileges. 

  For example, to start command prompt as an administrator
  following these steps:
  
  1. Press <kbd>Windows+R</kbd> to open the **Run** window. 
  
  2. Type `cmd` into the box. 
  
  3. Press <kbd>Ctrl+Shift+Enter</kbd> to run the command as an administrator.
  
{{site.alert.end}}
```

Reply:
looks great!